packages/wiz-ui-react/src/components/base/inputs/search-input/components/search-popup-panel.tsx

```diff
@@ -62,55 +64,62 @@ export const SearchPopupPanel: FC<Props> = ({
-                      styles.searchDropdownLabelStyle,
-                      isActive && styles.searchDropdownSelectingItemStyle
-                    )}
+                    className={styles.searchDropdownItemStyle}
+                    onMouseOver={() => setActiveValue(option.value)}
                   >
-                    {option.label}
-                    <WizIcon
-                      size="xl2"
-                      icon={WizIChevronRight}
-                      color={isActive ? "green.800" : "gray.500"}
+                    <div
+                      className={clsx(
+                        styles.searchDropdownLabelStyle,
+                        isActive && styles.searchDropdownSelectingItemStyle
+                      )}
+                    >
+                      {option.label}
+                      <WizIcon
+                        size="xl2"
+                        icon={WizIChevronRight}
+                        color={isActive ? "green.800" : "gray.500"}
+                      />
+                    </div>
+                  </div>
+                ) : (
+                  <div
+                    className={styles.searchDropdownCheckboxItemStyle}
+                    style={{ lineHeight: THEME.fontSize.xl3 }}
+                  >
+                    <WizCheckBox
+                      options={[
+                        {
+                          label: option.label,
+                          value: option.value,
+                          key: `${option.label}-${option.value}`,
+                        },
+                      ]}
+                      values={values}
+                      onChange={handleChangeValues}
                     />
                   </div>
-                </div>
-              ) : (
-                <div
-                  className={styles.searchDropdownCheckboxItemStyle}
-                  style={{ lineHeight: THEME.fontSize.xl3 }}
-                >
-                  <WizCheckBox
-                    options={[
-                      {
-                        label: option.label,
-                        value: option.value,
-                        key: `${option.label}-${option.value}`,
-                      },
-                    ]}
-                    values={values}
-                    onChange={handleChangeValues}
-                  />
-                </div>
-              )}
-              {i !== options.length - 1 && <WizDivider color="gray.300" />}
-            </div>
-          );
-        })}
+                )}
+                {i !== options.length - 1 && <WizDivider color="gray.300" />}
+              </div>
+            );
+          })
+        ) : (
+          <div className={styles.searchDropdownEmptyMessageStyle}>
+            {emptyMessage}
+          </div>
+        )}
       </div>
       {isOpen && (
         <SearchPopupPanel
-          options={activeOption.children}
+          options={activeOption.children ?? []}
```

Comment:
`activeOption.children` が `undefined` の場合はこの `SearchPopupPanel` は表示しないはずなので、こちらで Null 合体演算子を使って `options` に空配列を渡すのは厳密には正しい表現ではなくなってしまいます。

問題としては、`isOpen` は `activeOption.children` が `undefined` でないときにしか `true` にならないのにその型ガードが効いていないことなので、型ガードを効かせる一例としては、

`isOpen` を定義している場所を

```tsx
  const activeOptionChildren = activeOption?.children;
  const isOpen = activeOptionChildren !== undefined;
```

のようにして

```suggestion
          options={activeOptionChildren}
```

とすると型ガードが効くようになるので、Null 合体演算子で空配列を渡さなくてよくなります。